app/controllers/concerns/authentication.rb

```diff
@@ -27,24 +27,21 @@ def redirect_to_target_or_default
 
   def require_admin
     unless current_user
-      flash[:error] = "You must be logged in as an administrator to view this page."
-      redirect_to admin_login_url
+      redirect_to admin_login_url, alert: "You must be logged in as an administrator to view this page."
```

Comment:
The `:error` was in the admin, not in the front end which has a different layout.